lightning/src/routing/scoring.rs

```diff
@@ -252,7 +253,9 @@ pub type Scorer = ScorerUsingTime::<ConfiguredTime>;
 #[cfg(not(feature = "no-std"))]
 type ConfiguredTime = std::time::Instant;
 #[cfg(feature = "no-std")]
-type ConfiguredTime = time::Eternity;
+use util::time::Eternity;
+#[cfg(feature = "no-std")]
+type ConfiguredTime = Eternity;
 
```

Comment:
emm in line 255. there is also `#[cfg(feature = "no-std")]` so we will only import Eternity and ConfiguredTime set as Eternity.

I am not sure how to do the import and type declaration in one single statement. and it looks like there is cfg-if crate for providing a helpful macro to avoid duplicate cfg here? Is that what u meant?

Reply:
Ah, never mind. Forgot that you still would need to import the `util` module even if you fully-qualified as `util::time::Eternity` on line 258.